code/unclassified/src/TowerOfHanoi/Tower_of_hanoi.cpp

```diff
@@ -0,0 +1,43 @@
+//Algorithm of tower of Hanoi
+//START
+// Hanoi(disk_no, source, dest, aux)
+//   IF disk == 1, THEN
+//      move disk from source to dest             
+//   ELSE
+//      Hanoi(disk_no - 1, source, aux, dest)   : Step 1
+//      move disk from source to dest           : Step 2
+//      Hanoi(disk_no - 1, aux, dest, source)   :  Step 3
+//   END IF   
+//END Procedure
+//STOP
+#include<iostream>
+using namespace std;
+void TowerOfHanoi(int n, char from_rod, char to_rod, char aux_rod);
+int
+main()
+{
+    int n;
+    char source,destination,auxi;
+    cout<<"Enter no. of disk:";
+    cin>>n;
+    cout<<"Enter source rode name:";
+    cin>>source;
+    cout<<"Enter Destination rode name:";
+    cin>>destination;
+    cout<<"Enter Auxi. rode name:";
+    cin>>auxi;
+    TowerOfHanoi(n,source ,destination,auxi);
+   return 0;
+}
+
+void
```

Comment:
It should be void TowerOfHanoi in the same line

Reply:
Sorry,I was following c style.I will fix it .

Reply:
please,have a look now.